app.py

```diff
@@ -323,7 +322,7 @@ def swagger():
     :return:
     """
     # FIXME replace with one hosted here
-    req = requests.get("https://ga4gh.github.io/data-object-service-schemas/swagger/data_object_service.swagger.yaml")  # NOQA
+    req = requests.get("https://raw.githubusercontent.com/ga4gh/data-object-service-schemas/master/openapi/data_object_service.swagger.yaml")  # noqa
```

Comment:
In spirit, I appreciate trusting upstream like this. However, when master updates, clients asking this service for their OpenAPI description might not function with the service.

I think if you replaced with https://raw.githubusercontent.com/ga4gh/data-object-service-schemas/0.3.1/openapi/data_object_service.swagger.yaml then we would have some better guarantees. Added this issue https://github.com/DataBiosphere/dos-azul-lambda/issues/29 for presenting the subset of DOS from this lambda.



It points to two tangentially related issues.

First, clients should not require a swagger.json to be provided by a DOS. https://github.com/ga4gh/data-object-service-schemas/issues/114

Second, clients should be able negotiate the version information.

Reply:
You're right - thinking about it, this shouldn't have even been changed in this PR at all. (I changed it because the original link no longer exists, but I didn't realize that the old link was one that was manually updated.) I'll remove this change and address it in another PR.